.gitignore

```diff
@@ -6,6 +6,3 @@ node_modules
 .dev.vars
 
 .wrangler
-
-# shadcn components
-app/components/ui/**/*
```

Comment:
Happy to see this. I thought I was doing something wrong (I forced one new component in my branch to be able to build)